MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -460,26 +449,73 @@ internal bool HandlePacket(int packetId, Queue<byte> packetData)
                     default:
                         return true;
                 }
-            }
-            catch (Exception innerException)
-            {
-                if (innerException is ThreadAbortException || innerException is SocketException ||
-                    innerException.InnerException is SocketException)
-                    throw; //Thread abort or Connection lost rather than invalid data
-
-                throw new System.IO.InvalidDataException(
-                    string.Format(Translations.exception_packet_process,
-                        packetPalette.GetIncomingTypeById(packetId),
-                        packetId,
-                        protocolVersion,
-                        currentState == CurrentState.Login,
-                        innerException.GetType()),
-                    innerException);
-            }
+            //}
+            //catch (Exception innerException)
+            //{
+            //    //throw;
+            //    if (innerException is ThreadAbortException || innerException is SocketException ||
+            //        innerException.InnerException is SocketException)
+            //        throw; //Thread abort or Connection lost rather than invalid data
+
+            //    throw new System.IO.InvalidDataException(
+            //        string.Format(Translations.exception_packet_process,
+            //            packetPalette.GetIncomingTypeById(packetId),
+            //            packetId,
+            //            protocolVersion,
+            //            currentState == CurrentState.Login,
+            //            innerException.GetType()),
+            //        innerException);
+            //}
```

Comment:
I forgot to uncomment the try-catch here. Remember to uncomment it before release.